solidity/contracts/converter/BancorConverterUpgrader.sol

```diff
@@ -195,11 +199,21 @@ contract BancorConverterUpgrader is IBancorConverterUpgrader, ContractRegistryCl
             address connectorAddress = _oldConverter.connectorTokens(i);
             (virtualBalance, weight, isVirtualBalanceEnabled, , ) = _oldConverter.connectors(connectorAddress);
 
-            IERC20Token connectorToken = IERC20Token(connectorAddress);
-            _newConverter.addConnector(connectorToken, weight, isVirtualBalanceEnabled);
-
-            if (isVirtualBalanceEnabled)
-                _newConverter.updateConnector(connectorToken, weight, isVirtualBalanceEnabled, virtualBalance);
+            // Ether reserve
+            if (connectorAddress == address(0)) {
+                _newConverter.addETHReserve(weight);
+            }
+            // Ether reserve token

```

Comment:
`Ether reserve token` ->`EtherToken reserve`

Reply:
@yudilevi : We have 66 occurrences of `reserve token` and 0 occurrences of `token reserve` in our contract documentation. Are you sure?

Reply:
Not critical, going out soon anyway